components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/GraphQLAPIHandler.java

```diff
@@ -258,6 +259,10 @@ private boolean validatePayloadWithSchema(MessageContext messageContext, Documen
     private void handleFailure(MessageContext messageContext, String errorMessage) {
         OMElement payload = getFaultPayload(errorMessage);
         Utils.setFaultPayload(messageContext, payload);
+        Mediator sequence = messageContext.getSequence(APISecurityConstants.GRAPHQL_API_FAILURE_HANDLER);
+        if (sequence != null && !sequence.mediate(messageContext)){
```

Comment:
formatting issue

Reply:
Fixed in c4db271137c83d7f2a2a4c40a8c6248453a83817